test/test_von_mises.cpp

```diff
@@ -351,8 +370,13 @@ BOOST_AUTO_TEST_CASE( test_main )
   // (Parameter value, arbitrarily zero, only communicates the floating point type).
   test_spots(0.0F);   // Test float. OK at decdigits = 0 tolerance = 0.0001 %
   test_spots(0.0);    // Test double. OK at decdigits 7, tolerance = 1e07 %
+  // Check symmetry of PDF and CDF
+  test_symmetry(0.0F);
+  test_symmetry(0.0);
+
 #ifndef BOOST_MATH_NO_LONG_DOUBLE_MATH_FUNCTIONS
```

Comment:
I think we can now assume long double support; this macro was for older systems.